CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.22.0...master
+ - Fix regular expression for database url customization with `default_storage_engine`.
```

Comment:
As this is related to MySQL 5.7 introduction we could state it explicitly and also link JPERF-671. That said, we can do it when marking the next release as well.

Reply:
Oh, and this entry should go into `### Fixed` section.